src/Shared/NativeMethodsShared.cs

```diff
@@ -454,6 +454,12 @@ public SystemInformationData()
 
         #region Member data
 
+        static NativeMethodsShared()
```

Comment:
I'm not sure that helps, since a log of stuff calls into IsMono including anything that logs events.